social-compose/src/main/java/com/amity/socialcloud/uikit/community/compose/post/composer/AmityMediaAttachmentViewModel.kt

```diff
@@ -14,7 +14,11 @@ open class AmityMediaAttachmentViewModel : AmityBaseViewModel() {
     val postAttachmentPickerEvent get() = _postAttachmentPickerEvent
 
     private val _postAttachmentAllowedPickerType by lazy {
-        MutableStateFlow<AmityPostAttachmentAllowedPickerType>(AmityPostAttachmentAllowedPickerType.All)
+        MutableStateFlow<AmityPostAttachmentAllowedPickerType>(
+            // adding video attachments disabled temporarily due to lack of HEVC support
```

Comment:
[out of scope] [for the future]
I saw that Amity-iOS v4 added support for the HEVC format. We could test it, and if it works, ask Zumba about adding support for videos 👍